.github/workflows/main.yml

```diff
@@ -89,6 +89,26 @@ jobs:
     - name: Run check
       run: cargo hack check --feature-powerset --all-targets --examples --bins --tests --target ${{ matrix.target }}
 
+  Test_FreeBSD:
+    name: Test
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        target: ["x86_64-unknown-freebsd"]
+    steps:
+    - uses: actions/checkout@master
+    - name: Run check
+      uses: vmactions/freebsd-vm@v0.1.0
+      with:
+        prepare: |
+          pkg install -y curl
+          curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain nightly
```

Comment:
Two small things. Can you switch this to stable?

And can you add `--profile minimal`, it should speed up the download (a bit).